### ISSUE TITLE
Build STEM page in Drupal

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#11807]
## Issue Description
We need to build the STEM page in Drupal so we can share a preview with SMEs. Target deadline is 8.19.20.

---
## Tasks
- [x] Build page
- [x] Copy edit page and save as draft

## Acceptance Criteria
- [x] Page is built in Drupal and ready to share with SMEs

Preview link: preview-prod.vfs.va.gov/preview?nodeId=6860
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @RLHecht @MickinSahni. Note for sprint planning/assigning this work since I'm OOO Mon-Wed next week:

@bethpottsVADEPO is editing a Word doc of this page and will post here when it's ready. We need Drupal page built by 8/14 to stay on track with timeline for launch. Thanks!

**Comment by bethpottsVADEPO** [comment]
@RLHecht This is the same file I posted in #11804, just posting in the right ticket here!

[https://app.zenhub.com/files/133843125/33277a5f-e7ae-42d6-b22e-4487d66b3565/download](https://app.zenhub.com/files/133843125/33277a5f-e7ae-42d6-b22e-4487d66b3565/download)

**Comment by RLHecht** [comment]
@bethpottsVADEPO This page is now built. Preview link is listed at the top of this ticket. A few things:

1. We can't style the sext with the green button until we have a link to the online form. We also need to add the links to the online forms where we have the text. 

2. Under the question about private school rates, should we add a link to the Chapter 33 rates in the answer? 

3. Should the "Why does VA offer this scholarship?" come higher in the list of questions? I feel like since it explains why it's focused only on STEM degrees maybe it should?

**Comment by bethpottsVADEPO** [comment]
@RLHecht  Thanks! I'll let @DanielleThierryUSDSVA  handle these good issues you're raising. There are still some details to nail down. Thanks so much for getting this done to help us stay on track with the short timeframe.

**Comment by DanielleThierryUSDSVA** [comment]
Hello @RLHecht and @bethpottsVADEPO! Thanks so much for this page. I've looked at and talked through Beth's edits with her, and made edits to the edits and think this iteration is good to go.

@RLHecht We can close this ticket. But I wanted to first respond to your great notes/Qs above. Thank you!

_We can't style the text with the green button until we have a link to the online form. We also need to add the links to the online forms where we have the text._
- I styled the button with a link to the invision app form for now so we could show the smes the button.
- I had an open question with the product team about the paper form...they've confirmed there is no paper form (this is an online only application) so I removed that whole section of other ways to apply.

Under the question about private school rates, should we add a link to the Chapter 33 rates in the answer?
- Yes, thanks! I've added a link to the main rates page.

Should the "Why does VA offer this scholarship?" come higher in the list of questions? I feel like since it explains why it's focused only on STEM degrees maybe it should?
- This is a great question. I'd actually been leaning toward not including this (or the Q about the name of the scholarship) because they felt not benefit-focused enough...they were on the legacy page so I added them for thought. Your question speaks to, I think, the need for a little more detail up front and confirms the awkwardness of having that question shoved in at the end. So I removed those questions and instead added a mention of "high-demand careers" in the intro. Thank you!
